yb-voyager/cmd/metadataDB.go

```diff
@@ -97,3 +106,52 @@ func truncateTablesInMetaDb(exportDir string, tableNames []string) error {
 	}
 	return nil
 }
+
+func getTotalExportedEvents(runId string) (int64, int64, error) {
+	var totalCount int64
+	var totalCountRun int64
+	conn, err := sql.Open("sqlite3", getMetaDBPath(exportDir))
+	defer func() {
+		err := conn.Close()
+		if err != nil {
+			log.Errorf("failed to close connection to meta db: %v", err)
+		}
+	}()
+	query := fmt.Sprintf(`SELECT sum(num_total) from %s`, EXPORTED_EVENTS_STATS_TABLE_NAME)
+	err = conn.QueryRow(query).Scan(&totalCount)
+	if err != nil {
+		if !strings.Contains(err.Error(), "converting NULL to int64 is unsupported") {
+			return 0, 0, fmt.Errorf("error while running query on meta db -%s :%w", query, err)
+		}
+
+	}
+
+	query = fmt.Sprintf(`SELECT sum(num_total) from %s WHERE run_id = '%s'`, EXPORTED_EVENTS_STATS_TABLE_NAME, runId)
+	err = conn.QueryRow(query).Scan(&totalCountRun)
+	if err != nil {
+		if !strings.Contains(err.Error(), "converting NULL to int64 is unsupported") {
+			return 0, 0, fmt.Errorf("error while running query on meta db -%s :%w", query, err)
+		}
+	}
+
+	return totalCount, totalCountRun, nil
+}
+
+func getExportedEventsThroughputInLastNMinutes(runId string, n int) (int64, error) {
```

Comment:
Once the `MetaDB` class is merged in the `main` branch, we should make these functions as methods of the class.